src/qt/walletcontroller.cpp

```diff
@@ -124,8 +124,14 @@ WalletModel* WalletController::getOrCreateWallet(std::unique_ptr<interfaces::Wal
     }
 
     // Instantiate model and register it.
-    WalletModel* wallet_model = new WalletModel(std::move(wallet), m_client_model, m_platform_style, nullptr);
-    // Handler callback runs in a different thread so fix wallet model thread affinity.
+    WalletModel* wallet_model = new WalletModel(std::move(wallet), m_client_model, m_platform_style,
+                                                nullptr /* required for the following moveToThread() call */);
```

Comment:
In commit "qt: Improve comments in WalletController::getOrCreateWallet()" (2d6672ebd3095fb99e8b0fe0227697916665bb95)

Nice comments, these do a good job of explaining the context